pr-data.csv

```diff
@@ -2842,14 +2842,14 @@ https://github.com/apolloconfig/apollo,75f9950d5e1675dbb0617555c4502685ef4d4618,
 https://github.com/apolloconfig/apollo,75f9950d5e1675dbb0617555c4502685ef4d4618,apollo-biz,com.ctrip.framework.apollo.biz.service.NamespaceBranchServiceTest.testUpdateBranchGrayRulesWithUpdateOnce,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4599,
 https://github.com/apolloconfig/apollo,75f9950d5e1675dbb0617555c4502685ef4d4618,apollo-biz,com.ctrip.framework.apollo.biz.service.NamespaceBranchServiceTest.testUpdateBranchGrayRulesWithUpdateTwice,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4599,
-https://github.com/apolloconfig/apollo,ee028dd0981817b1b9f71d5a8e93ab1649271f75,apollo-configservice,com.ctrip.framework.apollo.configservice.service.AppNamespaceServiceWithCacheTest.testAppNamespace,ID,,,
+https://github.com/apolloconfig/apollo,ee028dd0981817b1b9f71d5a8e93ab1649271f75,apollo-configservice,com.ctrip.framework.apollo.configservice.service.AppNamespaceServiceWithCacheTest.testAppNamespace,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4998,
 https://github.com/apolloconfig/apollo,24062ad9b314da0fb44ccb941dcfd3da46a91fb7,apollo-configservice,com.ctrip.framework.apollo.configservice.service.ReleaseMessageServiceWithCacheTest.testWhenHasReleaseMsgAndHasRepeatMsg,ID,Accepted,https://github.com/apolloconfig/apollo/pull/2918,
 https://github.com/apolloconfig/apollo,75f9950d5e1675dbb0617555c4502685ef4d4618,apollo-configservice,com.ctrip.framework.apollo.configservice.service.ReleaseMessageServiceWithCacheTest.testWhenReleaseMsgSizeBiggerThan500,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4314,
-https://github.com/apolloconfig/apollo,53e6d7c202cdcc3c22b4fca9c29fe643340dcc82,apollo-core,com.ctrip.framework.apollo.core.utils.DeferredLoggerTest.testDeferredLog,ID,,,
 https://github.com/apolloconfig/apollo,e35ce8c013327b91d034b62d53eece055fa64970,apollo-client,com.ctrip.framework.apollo.internals.RemoteConfigLongPollServiceTest.testAssembleLongPollRefreshUrl,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4095,
 https://github.com/apolloconfig/apollo,e35ce8c013327b91d034b62d53eece055fa64970,apollo-client,com.ctrip.framework.apollo.internals.RemoteConfigLongPollServiceTest.testAssembleLongPollRefreshUrlWithMultipleNamespaces,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4095,
 https://github.com/apolloconfig/apollo,e35ce8c013327b91d034b62d53eece055fa64970,apollo-client,com.ctrip.framework.apollo.internals.RemoteConfigRepositoryTest.testAssembleQueryConfigUrl,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4083,
 https://github.com/apolloconfig/apollo,e35ce8c013327b91d034b62d53eece055fa64970,apollo-client,com.ctrip.framework.apollo.spring.config.ConfigPropertySourceTest.testGetPropertyNames,ID,Accepted,https://github.com/apolloconfig/apollo/pull/4082,
+https://github.com/apolloconfig/apollo-java,5344bc4f07288842bb850145bcf80a084bd94114,apollo-openapi,com.ctrip.framework.apollo.openapi.client.service.ItemOpenApiServiceTest.testFindItemsByNamespace,ID,,,
```

Comment:
How do we know that this new test is `ID`? You're removing another test that someone else thought may have been `ID` at some point. How did you check that this test fails due to the shuffling that NonDex does (and check multiple times to increase confidence it's not an `NOD` test)?

Reply:
How do we know that this new test is ID? 

I ran nondex on the entire submodule and this individual test as well. The test fails in both cases. I also figured out the reasons for flakiness and wrote a fix for the same. After this fix, nondex passes.

How did you check that this test fails due to the shuffling that NonDex
I ran IDFlakies for 20 rounds on the entire repo and got a successful build and 0 test failures.

Reply:
If you already have a fix, that's great. Don't just add a test with `,ID,,` then but open a PR and add `,ID,Opened,`.

Reply:
Done

Reply:
You need to add the URL of the real PR.

Reply:
I added a tentative PR to my forked repo: https://github.com/ThugJudy/apollo-java/pull/1 should I still add it?

Reply:
That change would not pass the checker. Please see email for more.